cdap-proto/src/main/java/co/cask/cdap/proto/metadata/lineage/FieldOperationInput.java

```diff
@@ -24,9 +25,13 @@
 import java.util.List;
 
 import javax.annotation.Nullable;
+
 /**
- * Represents input to the field operation.
+ * Represents input to the field operation. Input to the field operation
```

Comment:
Represents an input to a field operation:

- in case of a {@link co.cask.cdap.api.lineage.field.ReadOperation}, a list of datasets ({@link EndPoint});
- otherwise a list of {@link InputField}.

Reply:
fixed.